tools/choco/tinytex/tinytex.nuspec

```diff
@@ -11,7 +11,7 @@
     <projectUrl>https://yihui.org/tinytex/</projectUrl>
     <iconUrl>https://yihui.org/images/logo-tinytex.png</iconUrl>
     <copyright>Copyright 2020 Naveen M K,Yihui Xie</copyright>
-    <licenseUrl>https://github.com/yihui/tinytex/blob/master/tools/choco/tinytex/LICENSE</licenseUrl>
+    <licenseUrl>https://github.com/yihui/tinytex/blob/master/LICENSE</licenseUrl>
```

Comment:
This license only applies to the R package. Here I think what we need is the license for TinyTeX. I'm not sure if it's okay to point to https://tug.org/texlive/LICENSE.TL, but this document doesn't seem to be a license.

Reply:
For me, it doesn't hurt to point to texlive license https://tug.org/texlive/LICENSE.TL. Or what can be done here?

Reply:
I'm okay with that. My only concern is whether the chocolatey package reviewer would like it.

Reply:
We can explain him.